nestkernel/eprop_archiving_node_impl.h

```diff
@@ -138,14 +150,40 @@ EpropArchivingNode< HistEntryT >::erase_used_eprop_history()
     }
     else
     {
-      const auto it_eprop_hist_from = get_eprop_history( t );
-      const auto it_eprop_hist_to = get_eprop_history( t + update_interval );
-      eprop_history_.erase( it_eprop_hist_from, it_eprop_hist_to ); // erase found entries since no longer used
+      const auto it_eprop_hist_from_1 = get_eprop_history( t );
```

Comment:
What is the meaning of the `_1` and `_2` suffixes? Provide a comment.

Reply:
Fixed in https://github.com/nest/nest-simulator/pull/3207/commits/1d8d6fa46d056d08d15586ebc312d6641d9158f0